renderdoc/api/replay/replay_enums.h

```diff
@@ -1137,6 +1137,7 @@ enum class SpecialFormat : uint32_t
   D32S8,
   S8,
   YUV,
+  PVRTC
```

Comment:
The C# UI is around for a little while longer, so you need to update the enum here in [`Enums.cs`](https://github.com/baldurk/renderdoc/blob/master/renderdocui/Interop/Enums.cs#L205-L230) as well.